textscreen/txt_sdl.c

```diff
@@ -262,9 +262,24 @@ int TXT_Init(void)
 
     if (TXT_SDLWindow == NULL)
     {
+        int w, h;
+
+        if (font == &small_font || font == &large_font)
+        {
+            w = screen_image_w;
+            h = screen_image_h;
+        }
+        else
```

Comment:
What's left for the `else` case? Wouldn't it make it easier to understand if we checked for this instead? 

Also, `float` for `3 * x / 2`? :wink: